public/components/integrations/components/__tests__/__snapshots__/setup_integration.test.tsx.snap

```diff
@@ -50,27 +50,30 @@ exports[`Integration Setup Page Renders integration setup page as expected 1`] =
                         Object {
                           "connectionDataSource": "",
                           "connectionType": "index",
-                          "displayName": "sample Integration",
+                          "displayName": "[object Object] Integration",
```

Comment:
This looks wrong, can you take another look @Swiddis

Reply:
Woops, forgot to update a test -- fixed